ui/components/multichain/account-list-menu/account-list-menu.tsx

```diff
@@ -1,10 +1,18 @@
-import React, { useContext, useState } from 'react';
+import React, { useContext, useState, useMemo } from 'react';
 import PropTypes from 'prop-types';
 import { useHistory } from 'react-router-dom';
 import Fuse from 'fuse.js';
 import { useDispatch, useSelector } from 'react-redux';
+// eslint-disable-next-line import/no-duplicates
+import { BtcAccountType, EthAccountType } from '@metamask/keyring-api';
 ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
-import { KeyringClient } from '@metamask/keyring-api';
+import {
+  InternalAccount,
+  KeyringAccountType,
+  KeyringClient,
+  // eslint-disable-next-line import/no-duplicates
+} from '@metamask/keyring-api';
```

Comment:
Re-posting this one, cause he was not appearing (was outdated). Maybe we can do it like this:

```ts
import {
  EthAccountType
  ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
  InternalAccount,
  KeyringAccountType,
  KeyringClient,
  ///: END:ONLY_INCLUDE_IF
} from '@metamask/keyring-api';
///: BEGIN:ONLY_INCLUDE_IF(build-flask)
import { Caip2ChainId } from '@metamask/snaps-utils';
```